ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/provision/cucushift-installer-rehearse-aws-ipi-provision-chain.yaml

```diff
@@ -1,6 +1,7 @@
 chain:
   as: cucushift-installer-rehearse-aws-ipi-provision
   steps:
+  - ref: mco-conf-day2-add-mcoqe-robot-to-pull-secret
```

Comment:
@sergiordlr I think this day-2 step should be put after `ipi-install`, like `enable-qe-catalogsource`

Reply:
why ipi only? it should be available in both, ipi and upi, right?

Reply:
yes, it means the step should be executed when cluster is installed.